temporian/core/event_set_ops.py

```diff
@@ -1501,6 +1501,38 @@ def abs(
 
         return abs(self)
 
+    def __round__(self):
+        from temporian.core.operators.unary import round
+
+        return round(input=self)
+
+    def round(
```

Comment:
Missing adding the new op to the documentation: 
- missing `docs/src/reference/temporian/operators/unary/round.md` file (`docs/src/reference/temporian/operators/unary/abs.md` for example)
- missing adding the op to `docs/src/reference/index.md` (see `abs` definition for example, `round` should be added as a new line in the same table, respecting alphabetical order, making sure the prettier extension is working so that the markdown formatting is correct)

you can run `mkdocs serve -f docs/mkdocs.yml` to preview your changes to the docs locally